### ISSUE TITLE
[question] cli scripting interface for he3

### ISSUE THREAD

**Issue by chailampoon** [slashlos/He3#10]
Are there existing ways to communicate with he3 through cli? 

Advanced capabilities would be nice, but most relevant the ability to open a file/web url in an existing instance of he3. 

The obvious `open [url] -a /path/to/He3` seems to force a new window. 

I'm looking for an excuse to learn swift, so if not I might PR.

I'll probably just hack together some dreaded applescript first - let me know if you'd want me to link it in the docs.

**Comment by slashlos** [comment]
The scheme changed to he3lium <- he3-local <- helium.

But we're staying with he3lium until such time the name space can be recovered, at which point we'll support all.

So doing open <scheme>:///<some-things> should work.

**Comment by chailampoon** [comment]
I've come back to this and haven't been able to get it to work the way I'm looking for. Just for clarity, my goal is for some command to open a new url (web or local), in an **existing** instance and window of helium. 

At the moment
```
open [web-url] -a Helium.app
open [file-url] -a Helium.app
open h3lium://[web-url]
open h3lium://[file-url]
```
is able to open all destinations, but forces the opening of a new window. **Is there any modification to make it in the same window?**

---

Until now the best alternative I've found is using your (@slashlos) old fork of helium, where:
```
open helium://[web-url]
```
works perfectly, but:
```
open helium://[file-url]
```
does not.

---

I'd be very happy with a solution for both the helium fork and he3, though I'd prefer it for he3 as it's actively maintained and supports changing user agents + other features.

**Comment by slashlos** [comment]
I hesitate to stomp on the helium: scheme; if / when it's truly defunct I might adopt it, but until then these two (2) scheme will be supported: he3lium, he3., and a fix was made here appearing in 1.0.6 (build 8420+)
